### PR TITLE
[regression detector] Incorporate errors.txt into the issue alert

### PR DESCRIPTION
**Pull request by janeyx99**
This way, both runtime and perf regressions can be detected. Note that this only checks that the errors log file exists without reading from it.

This assumes that the errors.txt file will be generated by the benchmarks run step in the nightly and allows for users to tweak the path if desired.

DISCLAIMER: if the benchmarks fail due to an unrecoverable error and do not run til completion (such as OOMing), this will still NOT catch the issue, since it requires complete logs to be generated and exist.

### CODE REVIEW

File: regression_detector.py [hunk 1]
```diff
@@ -128,6 +141,8 @@ def process_regressions_into_gh_issue(regressions_dict, owner: str, output_path:
         "test_details": troubled_tests,
         "control_only_tests": control_only_tests,
         "treatment_only_tests": treatment_only_tests,
+        "runtime_regressions_msg": runtime_regressions_msg,
+        "github_run_id": os.environ.get("GITHUB_RUN_ID", ""),
```
Comment (xuzhao9) [inline regression_detector.py:145]: If `GITHUB_RUN_ID` is not set, we will print an invalid URL, maybe print something else when it is not set?

**Review by xuzhao9 — APPROVED** [review]
LGTM overall, see inline comments

File: regression_detector.py [hunk 2]
```diff
@@ -113,7 +118,15 @@ def process_regressions_into_gh_issue(regressions_dict, owner: str, output_path:
     control_commit = regressions_dict["control_env"]["pytorch_git_version"]
     treatment_commit = regressions_dict["treatment_env"]["pytorch_git_version"]
 
-    if troubled_tests == "" and control_only_tests == "" and treatment_only_tests == "":
+    runtime_regressions_msg = "No runtime errors were found in the " + \
+                              "new benchmarks run--you are all good there!"
+
+    errors_log_exists = Path(errors_path).exists()
+    if errors_log_exists:
```
[the remark this answers is not in the document]
Comment (janeyx99) [reply regression_detector.py:125]: A "No regressions found" line is printed if there are no perf nor runtime regressions later on. That should be sufficient, unless you're referring to something else?

File: regression_detector.py (hunk 1 above)
Comment (janeyx99) [reply regression_detector.py:145]: Okay!

File: regression_detector.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (xuzhao9) [reply regression_detector.py:125]: I mean if `runtime_regressions_msg` is `""`, in the GitHub issue we will see an empty line after "Runtime regressions found?", which might confuse people

[the remark this answers is not in the document]
Comment (janeyx99) [reply regression_detector.py:125]: Line 121 provides the alternate msg if no errors are found

[the remark this answers is not in the document]
Comment (xuzhao9) [reply regression_detector.py:125]: Sorry I missed that, I have no concerns then, thanks!

**Pull request closed without merging**